components/layout/block.rs

```diff
@@ -1491,7 +1491,7 @@ impl BlockFlow {
         debug_assert!(!self.is_inline_flex_item());
 
         // Compute the available space for us, based on the actual floats.
-        let rect = self.base.floats.available_rect(Au(0),
```

Comment:
You're totally right. I have looked to the details now. we only use `REPOSITION` to handle absolute positioning, not all out-of-flows as I thought. Also, it's handled in a later phase, so no need to do it here.